srcs/app/CGI/CGI.cpp

```diff
@@ -70,25 +70,28 @@ int CGI::ExecuteCGI(void) {
 	const std::string body = request_.GetBody();
 	std::fwrite(body.c_str(), 1, body.size(), fp);
 	std::rewind(fp);
-	SyscallWrap::pipeWr(fds_);
+	SyscallWrap::pipeWr(fds_, __FILE__, __FUNCTION__, __LINE__);
```

Comment:
I like these macros, I didn't know they existed :+1:

Reply:
I'm glad you learned about them